main.tf

```diff
@@ -34,7 +44,15 @@ module "obs_staging_vpc" {
   xchain_settings = {
     enabled      = true
     docker_image = local.xchain_indexer_staging_docker_image
-    config       = "staging"
+    config       = "config"
+    json_config = jsonencode({
+      omni_config = {
+        rpc_addr = local.omni_staging_rpc
+      }
+      external_chains = [
+        for x_chain in local.external_chains_staging : x_chain
+      ]
+    })
```

Comment:
What's the difference between `config`, json_config`, and `omni_config`?

Reply:
properly named, first two should now be clear from the naming. third one explained in the comment below.